src/manager/msNotificationManager.ts

```diff
@@ -184,7 +184,7 @@ export class MsNotificationDialog extends ModalDialog.ModalDialog {
                 {
                     label: _('Cancel'),
                     action: this._onCancelButtonPressed.bind(this),
-                    key: Clutter.Escape || Clutter.KEY_Escape,
+                    key: Clutter.KEY_Escape,
```

Comment:
This was meant for compatibility with < 3.34 Gnome Shell version do we assume remove the compatibility?

Reply:
You already specify on the project page that material-shell only supports gnome shell 3.34 or higher. So I see no reason to use the old variants. The typedefs only include the `KEY_X` variants, not the old stuff.